src/tools/compiletest/src/runtest.rs

```diff
@@ -1,5 +1,3 @@
-// ignore-tidy-filelength
```

Comment:
This was originally just me being curious, but since tidy likes it I'm just going to keep this change in and delete this.